packages/deployments/contracts/test/connext.spec.ts

```diff
@@ -167,18 +167,22 @@ describe("Connext", () => {
       ConnextUtils: connextUtils.address,
       RouterPermissionsManagerLogic: routerPermissionsManagerLogic.address,
     });
-    await originTm.initialize(originDomain, originBridge.address, originTokenRegistry.address, weth.address);
+    // TODO - Use real RelayerFeeRouter
+    await originTm.initialize(originDomain, originBridge.address, originTokenRegistry.address, weth.address, ZERO_ADDRESS);
 
     destinationTm = await deployContractWithLibs<Connext>("Connext", {
       AssetLogic: assetLogic.address,
       ConnextUtils: connextUtils.address,
       RouterPermissionsManagerLogic: routerPermissionsManagerLogic.address,
     });
+
+    // TODO - Use real RelayerFeeRouter
```

Comment:
will this happen in another subtask?

Reply:
Yes, this was updated in the integration tests PR https://github.com/connext/nxtp/pull/981